dialects/postgresql/postgresql-test-fixtures/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/PostgresqlTestFixtures.kt

```diff
@@ -0,0 +1,43 @@
+package app.cash.sqldelight.dialects.postgresql
+
+import java.io.File
+import java.util.Enumeration
+import java.util.jar.JarEntry
+import java.util.jar.JarFile
+
+object PostgresqlTestFixtures {
+  init {
+    loadFolderFromResources("fixtures_postgresql")
+  }
+  val fixtures = File("build/fixtures_postgresql").listFiles()
+    .filter { it.isDirectory }
+    .map { arrayOf(it.name, it) }
+}
+
+private fun Any.loadFolderFromResources(path: String) {
```

Comment:
Copied from https://github.com/AlecKazakova/sql-psi/blob/master/test-fixtures/src/main/kotlin/com/alecstrong/sql/psi/test/fixtures/FixturesTest.kt#L129

Ideally should be extracted somewhere